share/wake/lib/system/http.wake

```diff
@@ -29,36 +37,40 @@ tuple HttpRequest =
     Method: HttpMethod
     # The headers to be set on the request
     Headers: List HttpHeader
-    # The body of the request
-    Body: String
+    # A string of the contents of the request. Mutually exclusive with form data
+    Body: Option String
+    # Multipart form data. May be a list of strings or files. Mutually exclusive with body
+    FormData: Option (List HttpFormData)
 
 export tuple HttpResponse =
-    # The status code returned from the web server
-    StatusCode: Integer
+    # The status code returned from the web server. 
+    #
+    # Not all response have a status code so its not guarenteed to be set.
```

Comment:
haha I actually only added this because I hit it on some code I'm working on if you use curl to download a file on the local machine ala `curl -i file://foo/bar` you don't get the `HTTP/1.1 200` line

Reply:
Ah, right, not everything is HTTP!  Makes sense.